### PR TITLE
Keep last author in full author listing

### PR DESCRIPTION
**Pull request by rmarkello**
I'd accidentally dropped it while scraping the abstracts due to some poor regex-ing / testing. If you regenerate posters.json this should add the last authors back in—sorry about that!

### CODE REVIEW

File: tools/scrape_abstracts.py [hunk 1]
```diff
@@ -101,4 +102,4 @@ def loop_until_end(text, start, end, sep=''):
 
 # dump to json
 with open('abstract.json', 'w') as dest:
-    json.dump(abstracts, dest, indent=1, ensure_ascii=True)
+    json.dump(abstracts, dest, ensure_ascii=False)
```
Comment (yarikoptic) [inline tools/scrape_abstracts.py]: this is unrelated to authors fix...  I would just keep it all indented so diff is manageable to inspect please.

Comment (rmarkello) [reply tools/scrape_abstracts.py]: The initial (i.e., current) `abstract.json` wasn't indented because of an error on my part—that is, it's only one line. I just edited the script to match what I had accidentally uploaded initially, but I'll revert it and add the correctly-indented JSON if you'd like.

**Comment by rmarkello** [comment]
Should be all set—with pagination as-is do you think serving each abstracts' bag-of-words on the main site would still be too heavy?
